libs/designer/src/lib/ui/menuItems/runAfterMenuItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { MenuItem } from '@fluentui/react-components';
+import { bundleIcon, Copy24Filled, Copy24Regular } from '@fluentui/react-icons';
+import { LogEntryLevel, LoggerService } from '@microsoft/logic-apps-shared';
+import { useCallback } from 'react';
+import { useIntl } from 'react-intl';
+
+const RunAfterIcon = bundleIcon(Copy24Filled, Copy24Regular);
```

Comment:
THis is not a good choice for ICON for this, can we please pick something better. The "Dataflows" icon would work well here I think

Reply:
I cannot seem to find the 'DataFlows' icon in the fluent import files in LAUX. Does it have a specific name I should look for?

Alternatively, is there another icon you'd recommend I use instead?